packages/components/src/components/hds/badge/index.ts

```diff
@@ -121,7 +121,7 @@ export default class HdsBadgeComponent extends Component<HdsBadgeSignature> {
    * @type {string|null}
    * @default null
    */
-  get icon(): FlightIconSignature['Args']['name'] | null {
+  get icon(): HdsIconSignature['Args']['name'] | null {
```

Comment:
should probably be `undefined`
```suggestion
  get icon(): HdsIconSignature['Args']['name'] | undefined {
```

Reply:
Probably no reason to keep the getter then, right?

Reply:
right, good point, we may be able to get rid of it